### ISSUE TITLE
iCal feed of events you've RSVP'd to

### ISSUE THREAD

**Issue by strugee** [kokeshii/RSVPBot#72]
It would be _sick_ if RSVPbot was able to give me an iCal feed I could plug into my calendar so stuff automatically shows up when I `rsvp yes`.

Related: #28

**Comment by alexandrinaw** [comment]
Agreed. The google calendar integration does invite the email addresses of folks who've RSVP'd yes to an event, so they show up some people's calendars, but not everyone :/. 

If this can be done quickly it might be worth adding to RSVPBot, but RC is currently working on a super awesome unified events system that will include this feature, so in a few months an RSVPBot ical feed might become obsolete.
